README.md

```diff
@@ -93,7 +93,7 @@ func (s *Session) Rcpt(to string) error {
 	return nil
 }
 
-func (s *Session) Data(r io.Reader) error {
+func (s *Session) Data(r io.Reader, sc smtp.LMTPStatusCollector) error {
```

Comment:
Since this is only for LMTP, I wonder if there's a way to make it optional…

Reply:
Only if we would split out lmtp completely. But its not hard for SMTPD to fill that interface, just update the data function definition

Reply:
If we are really serious about backward-compatibility - we can add an interface like that:
```go
type LMTPSession interface {
  LMTPData(r io.Reader, sc smtp.LMTPStatusCollector) error
}
```
and assert for it.

But I agree with mschneider82, it is pretty easy to fix existing code and we are at 0.x.